scripts/Doxyfile

```diff
@@ -848,7 +848,8 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = README.md
+EXCLUDE                = README.md \
+                         ../include/ur_print.hpp
```

Comment:
Just occurred to me that these APIs won't have doxygen docs. We should eventually address this.

Reply:
Added generation of docs for all operators and a function from ::extras.